logging/logrus/registry.go

```diff
@@ -229,3 +231,8 @@ func (lr *logRegistry) getLoggerFromMapping(logger string) *Logger {
 	panic("cannot cast log value to Logger obj")
 
 }
+
+// HookConfigs stores hook configs from log manager
+func (lr *logRegistry) AddHook(hook logrus.Hook) {
+	lr.hooks = append(lr.hooks, hook)
```

Comment:
Where is the hooks field even used? It should be applied to all new loggers created in this registry.